### ISSUE TITLE
RetroArch takes keyboard input while not focused if in fullscreen

### ISSUE THREAD

**Issue by LinAGKar** [libretro/RetroArch#9069]
## Description

RetroArch takes keyboard input while not in focus if in fullscreen. This happens with all input drivers, except udev, which doesn't take keyboard input at all.

### Expected behavior

It should not take input while in background.

### Actual behavior

It takes keyboard in input while in fullscreen in background.

### Steps to reproduce the bug

1. Open RetroArch
2. Switch to fullscreen
3. Switch to a different virtual desktop, or open a window in front of RetroArch
4. Press a key, e.g. `f`

### Version/Commit

- RetroArch: 1.7.7

### Environment information

- OS: openSUSE Tumbleweed 20190702
- DE: KDE Plasma 5.16.2

**Comment by i30817** [comment]
Udev needs the user in the input group iirc.

**Comment by LinAGKar** [comment]
Adding my user to the input group makes the udev keyboard input work, and it exhibits the same behavior. Though controller input via udev already worked.

**Comment by i30817** [comment]
Yeah, i meant keyboard. It's a good idea anyway, because without that if you try to run RA on a login TTY there is a risk of a softlock when the search widget pops up, where your only resource is REISUB (because retroarch disables VT-switch because of the DRM architecture).

**Comment by orbea** [comment]
I can't reproduce this, maybe its related to KDE?

**Comment by LazyBumHorse** [comment]
Quoting SolitudeSF from #9248:
> i found that `Start in fullscreen` option turned on is causing this.
> i should add that it doesnt matter if window is fullscreen or not. it only matters that it starts in fullscreen.

Can you reproduce this?
@orbea Are you starting RA in windowed mode?

**Comment by LinAGKar** [comment]
For me it stops grabbing input when it switches to windowed, and starts grabbing it when it switches to exclusive fullscreen (but not windowed fullscreen), regardless of what it started in. Very similar to #9248, but not quite the same, as for @SolitudeSF the problem sticks after he leaves fullscreen, while for me it doesn't.

**Comment by Scall** [comment]
The issue can also be worked around by using Wayland.
